Lib/unittest/mock.py

```diff
@@ -1085,15 +1085,15 @@ def _get_child_mock(self, /, **kw):
         return klass(**kw)
 
 
-    def _calls_repr(self, prefix="Calls"):
+    def _calls_repr(self, prefix="Calls", display_empty=False):
         """Renders self.mock_calls as a string.
 
         Example: "\nCalls: [call(1), call(2)]."
 
         If self.mock_calls is empty, an empty string is returned. The
         output will be truncated if very long.
         """
-        if not self.mock_calls:
+        if not self.mock_calls and not display_empty:
```

Comment:
What happens if we don't have this `display_empty` and instead just remove this whole `if` block?

Reply:
In `assert_called_once()` it will display:
```
Expected 'mock' to have been called once. Called 0 times.
Calls: []
```
`Calls: []` is redundant here, because it already shows that it was called 0 times.